homework.md

```diff
@@ -44,61 +44,63 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A scripting language used to create dynamic, interactive web pages
 
 ## Question 8
 
-What three problem does Git & Github solve?
+What three problems does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. It makes file sharing easy
+2. Tracking changes made to a file is simple
+3. It facilitates collaboration and team work
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```

Comment:
RE: Question 9
Hi Biruk,

The answer for the question is correct,. 
Could you give more details about what happens in the process of a repository. 

Aaron

Reply:
Hi Aaron,
I'll check it and try to amend it. Thanks for the feedback.